Templates/MultiGemRepo/template.json

```diff
@@ -0,0 +1,42 @@
+{
+    "template_name": "MultiGemRepo",
```

Comment:
If I am not mistaken this is meant for someone to copy the template to use for their own repos. If so, minor nitpick, but we should change the name to MultiGemRepo_Template. This will make it clear if someone uses the template but doesn't change the name.

Reply:
The user is meant to instantiate this template to crate their gem repo.
It is not meant to be copied by the gem maintainers.

Reply:
The `template_name` field is just the name of the template.  The user wouldn't copy the template.json file for their own repos.  The name of the repo is in `repo_name` field in `repo.json` and is automatically set when the template is used to create a new repo.